tests/foreman/cli/test_oscap.py

```diff
@@ -204,8 +193,9 @@ def test_negative_info_scap_content(self):
         with pytest.raises(CLIReturnCodeError):
             Scapcontent.info({'id': invalid_scap_id})
 
+    @pytest.mark.parametrize('title', **parametrized(valid_data_list()))
```

Comment:
:+1:

Reply:
Also, Make sure you add the `:parameterized: yes` token to the test docstring.

Reply:
Added `:parameterized: yes`